### ISSUE TITLE
Axis ticks/boxes of plots inside trees do not fit if numbers get to large

### ISSUE THREAD

**Issue by jmozmoz** [mljar/supertree#29]
This is a separate bug report for formating axis ticks. The original issue is https://github.com/mljar/supertree/issues/28

If the number elements get to big, the axis ticks do not fit into the boxes inside plots. See this example>

```python

from sklearn.tree import DecisionTreeClassifier
from sklearn.datasets import load_iris
from supertree import SuperTree # <- import supertree :)
import numpy as np

# Load the iris dataset
iris = load_iris()
X, y = np.repeat(iris.data, 100, axis=0), np.repeat(iris.target, 100, axis=0)

# Train model
model = DecisionTreeClassifier()
model.fit(X, y)

# Initialize supertree
super_tree = SuperTree(model, X, y, iris.feature_names, iris.target_names)

# show tree in your notebook
super_tree.show_tree()
```

![grafik](https://github.com/user-attachments/assets/bcc55e07-ef72-4f60-96b4-cb31d53a2f15)

The example demonstrates that the boxes of the graphs seem to be hard coded and do not consider the axis "ticks" (matplotlib speak). Is there a way to e.g. enforce using "scientific" formating of the numbers of the axis? Or change the size of the boxes dynamically?

**Comment by Marchlak** [comment]
Now large numbers should extend toward the chart instead of overflowing beyond the background.
